docs/quick_start.md

```diff
@@ -94,30 +148,33 @@ $HOME
                       |             |--- add_sub
                       |                     |--- gpu_mem_v_latency.png
                       |                     |--- throughput_v_latency.png
-                      | 
+                      |
                       |--- results
-                      |       |--- metrics-model-inference.csv 
-                      |       |--- metrics-model-gpu.csv 
+                      |       |--- metrics-model-inference.csv
+                      |       |--- metrics-model-gpu.csv
                       |       |--- metrics-server-only.csv
                       |
                       |--- reports
-                              |--- summaries 
+                              |--- summaries
                               .        |--- add_sub
                               .                |--- result_summary.pdf
 ```
 
-## Generate a detailed report
+## `Step 5:` Generate a Detailed Report
+
+---
 
 Model analyzer's report subcommand allows you to examine the performance of a
 model config variant in detail. For example, it can show you the latency
 breakdown of your model to help you identify potential bottlenecks in your model
-performance. The detailed reports also contain other configurable plots and a
+performance.<br><br>
+The detailed reports also contain other configurable plots and a
 table of measurements taken of that particular config. You can generate a
 detailed report for the two `add_sub` model configs `add_sub_config_default` and
 `add_sub_config_0` using:
 
 ```
-$ model-analyzer report --report-model-configs add_sub_config_default,add_sub_config_0 -e analysis_results 
+$ model-analyzer report --report-model-configs add_sub_config_default,add_sub_config_0 -e analysis_results
 ```
 
 This will create directories named after each of the model configs under
```

Comment:
On the next line: Need a backslash after the dot:
`./analysis_results/reports/detailed`

Reply:
Fixed